api/grpc/logging_interceptor.go

```diff
@@ -0,0 +1,39 @@
+// Any changes to this file should be considered for its counterpart:
+// pkg/clairify/server/middleware/log.go
+
+package grpc
+
+import (
+	"context"
+	"time"
+
+	"github.com/sirupsen/logrus"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/peer"
+	"google.golang.org/grpc/status"
+)
+
+func loggingUnaryServerInterceptor() grpc.UnaryServerInterceptor {
+	return func(ctx context.Context, req interface{}, info *grpc.UnaryServerInfo, handler grpc.UnaryHandler) (interface{}, error) {
+		peerInfo, exists := peer.FromContext(ctx)
+		if !exists {
+			return nil, status.Error(codes.InvalidArgument, "unable to parse peer information from request context")
+		}
+
+		logrus.WithFields(map[string]interface{}{
+			"Method": info.FullMethod,
+		}).Infof("Received gRPC request from %v", peerInfo.Addr)
```

Comment:
I'm not an expert on grpc-gateway - but looks [like it supports](https://github.com/grpc-ecosystem/grpc-gateway/blob/f238f782038e2f05e1c238868b07db477fdf3782/runtime/context_test.go#L116) adding `X-Forwarded-For` header to the context 

Guessing should be able to extract the `Metadata` from the context, something like:
```
addrs := metadata.ValueFromIncomingContext(ctx, "X-Forwarded-For")
```
And then append the `addrs` to this log

Reply:
doesn't seem like we can rely on this:

```
{"Event":"Received gRPC request from unknown","HEADERS":{":authority":["localhost:8443"],"content-type":["application/grpc"],"user-agent":["grpc-go/1.57.0"]},"Level":"info","Location":"logging_interceptor.go:28","Method":"/scannerV1.ImageScanService/GetImageComponents","Time":"2023-08-23 22:25:36.923706"}
```

Going to revert back to the original but leave out the peer's address because it added no value